packages/linux/patches/linux-010_fix_device_table_aliases.patch

```diff
@@ -0,0 +1,12 @@
+diff -ur a/include/linux/module.h b/include/linux/module.h
```

Comment:
Pls, remove linux patch as I fixed it in amlogic-3.14 kernel.

Reply:
> Pls, remove linux patch as I fixed it in amlogic-3.14 kernel.

Done!